README.md

```diff
@@ -0,0 +1,160 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 OOP Catalog of my things ](#-catalog-of-my-things-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [🚀 demo video](#live-demo)
+  - [� Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog of my things <a name="about-project"></a>
+
+**Catalog of my things** This console app helps users keep track of their possessions like books, music albums, movies, and games based on a UML class diagram. It stores data in JSON files and has a corresponding database with tables that match the program's class structure.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Server</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/getting_started.html">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- [keep a record of different types of things you own]
+- [Store the data in JSON files]
+- [Add items you own]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
Good work on the presentation, since this has to be a presentation that your future Recruiters can see, please make the following change
- Kindly record a new video with a piece of code that you find interesting for each one of you. 
- Remember to use the time accordingly, in this video, you have to stop at 04:58 and a mate was talking. That doesn't like good in a presentation. Kindly make a better end on the video too. It's ok if the video overextends a little more than 5 minutes since you are three people in the presentation. You can practice it before recording the video 💯